exercises/concept/ticket-please/tests/Tests.elm

```diff
@@ -0,0 +1,277 @@
+module Tests exposing (task1, task2, task3, task4)
+
+import Expect
+import Test exposing (Test, describe, test)
+import TicketPlease exposing (..)
+
+
+task1 : Test
```

Comment:
I think there is a way that the test runner wants these tests to be defined, and I think it is to have them all in the same function, and use `describe` to separate out the tests for the various cases. Also, just for information, the test runner doesn't support use of fuzz testing (which isn't an issue for the PR, but useful to know)